server/routes.go

```diff
@@ -136,10 +136,36 @@ func (r *routesImpl) SetDefaultRoute(backend string) {
 	}).Info("Using default route")
 }
 
+var simplifySRV bool
+
+func SimplifySRV(srvEnabled bool) {
+	simplifySRV = srvEnabled
+}
+
```

Comment:
honestly, yeah, this actually makes an bit more sense now that this has been pointed out, its an config table that already exists, why should I define an nother loose variable? when I can bundle it there so that it may be re-used in other functions that may end up needing to detect it later down the line (some server plugins use UDP on the same port as the MC server for things like voicechat)